components/product-listing/wishlistPage/wishlist.css

```diff
@@ -0,0 +1,282 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--lt-cream);
+    font-family: 'Arima Madurai', cursive;
+
+}
```

Comment:
This again is redandant. add it in reset files

Reply:
yeah i'll do that tomorrow